src/methods/malApi/anime/index.ts

```diff
@@ -22,7 +22,9 @@ export class MalAnime {
     q: string,
     fields?: AnimeFields<T> | null,
     limit?: number | null,
-    offset?: number | null
+    offset?: number | null,
+    nsfw? : boolean | true
```

Comment:
What does `boolean | true` mean? I would expect `boolean` type to already include `true` case.